src/gui/guiChatConsole.cpp

```diff
@@ -219,6 +234,11 @@ void GUIChatConsole::reformatConsole()
 	s32 rows = m_desired_height / m_fontsize.Y - 1; // make room for the input prompt
 	if (cols <= 0 || rows <= 0)
 		cols = rows = 0;
+
+	if (m_scrollbar_enabled) {
+		m_scrollbar->setRelativePosition(core::rect<s32> (m_screensize.X - 32, 0, m_screensize.X, m_height));
```

Comment:
needs to take display density and perhaps gui_scaling into account (also deduplicate it)

Reply:
moved into updateScrollbar().